File: All-languages/array/arrays-maximum-value/maximum.js
var numbers = [1,2,34,65,3223,3434]

var maximum = numbers[0]

for (var i=0;i<=numbers.length;i++) {
	var num = numbers[i]
	if (num > maximum) {
		maximum = num
	}
}
console.log(maximum)

File: All-languages/array/arrays-maximum-value/maximum.go
package main
import "fmt"

func main() {

	arrays := []int{1,2,232,454,2323,354,5656,434354}
	maximum := arrays[0]

	for _,num := range arrays {
		if num > maximum {
			maximum = num
		}
	}
	fmt.Println(maximum)
}
